core/execution/collateral.go

```diff
@@ -80,17 +80,21 @@ func (m *Market) transferMarginsAuction(ctx context.Context, risk []events.Risk,
 	return nil
 }
 
-func (m *Market) transferRecheckMargins(ctx context.Context, risk []events.Risk) error {
+func (m *Market) transferRecheckMargins(ctx context.Context, risk []events.Risk) {
 	if len(risk) == 0 {
-		return nil
+		return
 	}
 	mID := m.GetID()
 	evts := make([]events.Event, 0, len(risk))
 	for _, r := range risk {
+		var tr *types.LedgerMovement
 		responses := make([]*types.LedgerMovement, 0, 1)
 		tr, closed, err := m.collateral.MarginUpdateOnOrder(ctx, mID, r)
 		if err != nil {
-			return err
+			m.log.Warn("margin recheck failed",
+				logging.MarketID(m.GetID()),
+				logging.PartyID(r.Party()),
+				logging.Error(err))
```

Comment:
@EVODelavega please confirm that you're happy with this. This seemed like a best choice to me as it still logs a warning (possibly more than one now, before it was an umbrella warning here https://github.com/vegaprotocol/vega/blob/develop/core/execution/market.go#L478C10-L484.

Could also just `log.Debug` if `closed!=nil` (it just means somone's marked for closeout but it cannot be done yet as there's not enough volume, so nothing sinister) and leave `Warn` for other cases (there shouldn't be any).

Reply:
yeah, that's fair enough